app/src/main/res/layout/fragment_credit_card_editor.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:orientation="vertical">
+
+    <!-- Card Number -->
+    <TextView
```

Comment:
I see 2 a11y issues on this current layout:
- the touch target of the input layouts is smaller than the recommended 48dp.
- using another view as the hint for the input layouts leads to them being read as "Edit Box".
Don't know if just using `labelFor` on the TextViews is enough. This would still mean Talkback would see them as 2 separate targets. Using a parent layout for them would probably help or having them in a new custom view. ?

Reply:
Let's address this in a follow up to avoid the flaky test from a re-pushing to this PR.